src/components/EmbeddedContent.tsx

```diff
@@ -1,5 +1,26 @@
 import React from "react";
 
+interface AspectRatioProps {
+  aspectRatio: number;
+  children: any;
+}
+
+const AspectRatio = ({ aspectRatio, children }: AspectRatioProps) => (
```

Comment:
Can we make this its own export in its own file?

Reply:
Sure thing. I'll update after some calls this morning.

Reply:
Revised